lms/djangoapps/student_account/views.py

```diff
@@ -125,7 +125,7 @@ def login_and_registration_form(request, initial_mode="login"):
         'responsive': True,
         'allow_iframing': True,
         'disable_courseware_js': True,
-        'disable_footer': True,
+        'disable_footer': not get_themed_value('ENABLE_COMBINED_LOGIN_REGISTRATION_FOOTER', settings.FEATURES['ENABLE_COMBINED_LOGIN_REGISTRATION_FOOTER']),
```

Comment:
Please use `settings.FEATURES.get('ENABLE_COMBINED_LOGIN_REGISTRATION_FOOTER', False)` for the default value.